README.md

```diff
@@ -139,6 +153,112 @@ You can find an example of the SQL file at `tests/sql_script.sql`.
 
 Default: `null`
 
+Type: `str`
+
+### `mssql_enable_sql_agent`
+
+Set this variable to `true` or `false` to enable or disable the SQL agent.
+
+Default: `null`
+
+Type: `bool`
+
+### `mssql_install_fts`
+
+Set this variable to `true` or `false` to install or remove the
+`mssql-server-fts` package that provides full-text search.
+
+Default: `null`
+
+Type: `bool`
+
+### `mssql_enable_ha`
+
+Set this variable to `true` or `false` to install or remove the
+`mssql-server-ha` package and enable or disable the `hadrenabled` setting.
+
+Default: `null`
+
+Type: `bool`
+
+### `mssql_tune_for_fua_storage`
+
+Set this variable to `true` or `false` to enable or disable settings that
+improve performance on hosts that support Forced Unit Access (FUA) capability.
+
+Only set this variable to `true` if your hosts are configured for FUA
+capability.
+
+When set to `true`, the role applies the following settings:
+
+* Set the `traceflag 3979 on` setting to enable trace flag 3979 as a startup
+parameter
+* Set the `control.alternatewritethrough` setting to `0`
+* Set the `control.writethrough` setting to `1`
+
+When set to `false`, the role applies the following settings:
+
+* Set the `traceflag 3982 off` parameter to disable trace flag 3979 as a
+startup parameter
+* Set the `control.alternatewritethrough` setting to its default value `0`
+* Set the `control.writethrough` setting to its default value `0`
+
+For more details, see SQL Server and Forced Unit Access (FUA) I/O subsystem
+capability at <https://docs.microsoft.com/en-us/sql/linux/sql-server-linux-performance-best-practices?view=sql-server-ver15>.
+
+Default: `null`
+
+Type: `bool`
+
+### `mssql_tls`
+
+Use this variable to configure MSSQL to encrypt connections using TLS
+certificates. You must have the TLS certificate and private key on the Ansible
+control node.
+
+When you use this variable, the role copies TLS cert and private key files to
+MSSQL and configures MSSQL to use these files to encrypt connections.
+
+Default: `null`
+
+Type: `dict`
+
+#### `enable`
+
+Set to `true` or `false` to enable or disable TLS encryption. When set to
+`true`, the role uploads certificate and private key files to the host and
+configures mssql to encrypt connections using these files. When set to `false`,
+the role configures mssql to not use TLS encryption. The role does not remove
+the existing certificate and private key files if this variable is set to
+`false`.
+
+Default: `null`
+
+Type: `bool`
+
+#### `cert`
+
+Path to the certificate file to copy to MSSQL.
```

Comment:
This is the path and filename?

Reply:
Ansible accepts both the filename, given that the file resides in the same directory as the ansible-playbook, and the full path. Do you suggest rewording this?

Reply:
> Do you suggest rewording this?

No - I just checked and this wording is consistent with other roles